comp/otelcol/extension/impl/extension.go

```diff
@@ -76,7 +76,7 @@ func (ext *ddExtension) Start(_ context.Context, host component.Host) error {
 
 	extensions := host.GetExtensions()
 	for extension := range extensions {
-		extractor, ok := supportedDebugExtensions[extension.String()]
+		extractor, ok := supportedDebugExtensions[extension.Type().String()]
```

Comment:
I assume this `Type()` call strips what follows the `/` in the extension name?

Reply:
Yes, a component ID comprises of a [Type and a Name](https://github.com/open-telemetry/opentelemetry-collector/blob/component/v0.107.0/component/identifiable.go#L19-L22). The Type is what is before the /, and the name is what is after the /.